src/library_sockfs.js

```diff
@@ -383,6 +383,10 @@ addToLibrary({
           return sock.pending.length ? ({{{ cDefs.POLLRDNORM }}} | {{{ cDefs.POLLIN }}}) : 0;
         }
 
+        if (sock.error) {
+          return {{{ cDefs.POLLOUT }}};
+        }
```

Comment:
Can you explain and maybe add a comment, why all errors are treated as POLLOUT ("writing is now possible")? Are no other errors possible?

Reply:
I added a comment.

While it is possible I'm casting too wide of a net here and other times that `sock.error` are set do not warrant being marked as readable, it would require more state tracking to determine that the source of the error was a call to `connect` since as far as I can tell we don't track the current state of the connection.

Reply:
Turns out I did need the extra state in order to be more precise here.  Cleaner now.